src/arch/arm/64/kernel/vspace.c

```diff
@@ -551,41 +551,51 @@ BOOT_CODE void activate_kernel_vspace(void)
 BOOT_CODE void write_it_asid_pool(cap_t it_ap_cap, cap_t it_vspace_cap)
 {
     asid_pool_t *ap = ASID_POOL_PTR(pptr_of_cap(it_ap_cap));
-    ap->array[IT_ASID] = (void *)(pptr_of_cap(it_vspace_cap));
-    armKSASIDTable[IT_ASID >> asidLowBits] = ap;
+    asid_map_t asid_map = asid_map_asid_map_vspace_new(
 #ifdef CONFIG_ARM_SMMU
-    vspace_root_t *vtable = ap->array[IT_ASID];
-    vtable[VTABLE_SMMU_SLOT] = vtable_invalid_smmu_new(0);
+                              /* bind_cb: Number of bound context banks */
+                              0,
+#endif
+                              /* vspace_root: reference to vspace root page table object */
+                              (word_t)cap_vtable_root_get_basePtr(it_vspace_cap)
+#ifdef CONFIG_ARM_HYPERVISOR_SUPPORT
+                              /* stored_hw_vmid, stored_vmid_valid: Assigned hardware VMID for TLB. */
+                              , 0, false
 #endif
+                          );
+    ap->array[IT_ASID] = asid_map;
+    armKSASIDTable[IT_ASID >> asidLowBits] = ap;
 }
 
 /* ==================== BOOT CODE FINISHES HERE ==================== */
 
-static findVSpaceForASID_ret_t findVSpaceForASID(asid_t asid)
+asid_map_t findMapForASID(asid_t asid)
 {
-    findVSpaceForASID_ret_t ret;
-    asid_pool_t *poolPtr;
-    vspace_root_t *vspace_root;
+    asid_pool_t        *poolPtr;
```

Comment:
```suggestion
    asid_pool_t *poolPtr;
```

(at least as far as I can see in the diff, there are no other declarations to align with any more)

Reply:
Yea, it came in with a cherry-picked commit and I assumed the style tool would fix it up.....